conditional/util/flask.py

```diff
@@ -18,16 +19,14 @@
 from conditional import db
 
 
-def render_template(request, template_name, **kwargs):
-    user_name = request.headers.get('x-webauth-user')
-
-    # TODO maybe use the webauth request decorator
+@get_username
+def render_template(template_name, username=None, **kwargs):
```

Comment:
Why rename `user_name` to `username`? Or at least this seems like something that would fit better into the style commit since it doesn't appear to make a functional difference. Not renaming this would dramatically reduce the size of your commit.

If you do decide to move this to a style commit please move the numerous changes that you've made to the other files in this commit there as well.

Reply:
I renamed it to username, because they were inconsistent throughout. Also `username` was the first one I found, so the next two `user_name`s felt wrong. It was quite far in before I realized `username` was the odd one out. However `username` felt more proper, given it's often used as a real term

Reply:
See my previous comment. We should just include this in the wrapper and then we don't have to worry about it.